common/events/player_event_logs.cpp

```diff
@@ -123,21 +134,466 @@ void PlayerEventLogs::ProcessBatchQueue()
 
 	BenchTimer benchmark;
 
+	EtlQueues etl_queues{};
```

Comment:
Looking at all of this in ProcessBatchQueue and in our deletions, I think we should wrap these functions in thread calls so we don't tie up the main thread. 

This also means that we should create a separate database connection just for processing player events because what happens is even if we had all of this in another thread its going to be lock contended with database locks on the main thread.

World needs to be kept as lightweight as possible in the main thread and this certainly adds weight if anything starts to take more than 500ms which it most certainly could here.

Same would go for using QS if folks use QS to process. 

I may end up taking this one on.